lib/content-services/src/lib/permission-manager/components/add-permission/add-permission-panel.component.html

```diff
@@ -34,7 +35,9 @@
             class="adf-permission-result-list"
             [class.adf-permission-result-list-search]="searchedWord.length === 0">
     <ng-template let-data>
-        <mat-selection-list class="adf-permission-result-list-elements" (keydown.control.a)="onSelectionChange()"
+        <mat-selection-list class="adf-permission-result-list-elements"
+                            title="{{'PERMISSION_MANAGER.ADD-PERMISSION.USER-GROUP-LIST' | translate}}"
+                            (keydown.control.a)="onSelectionChange()"
```

Comment:
can you see if this one can be removed? binding "ctrl+a" makes no sense here, and it's the only place ever used